LiteCore/Database/PrebuiltCopier.hh

```diff
@@ -0,0 +1,16 @@
+//
+//  PrebuiltCopier.hpp
+//  LiteCore
+//
+//  Created by Jim Borden on 2017/08/15.
+//  Copyright © 2017 Couchbase. All rights reserved.
+//
+
+#pragma once
+#include "FilePath.hh"
```

Comment:
You don't need to include these headers; just declare them with `class FilePath;` etc. inside the `namespace` block below.